File: Go/sum_range/sum_range.go
package main

import "fmt"

func sumRange(x int, y int) int {
	p := 0
	for i := x; i <= y+1; i++ {
		fmt.Println(p)
		p += i
	}
	return p
}

func main() {
	sumRange(10, 20)
}

File: Java/sumRange.java
class SumRange {
    public static void main(String[] args) {
        System.out.println(sumRange(22, 30));
    }

    public static int sumRange(int x, int y) {
        int p = 0;

        for (int i = x; i <= y+1; i++) {
            p += i;
        }

        return p;
    }    
}
